curvefs/src/mds/schedule/operatorFactory.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * @Project: curve
+ * @Date: 2021-11-8 11:01:48
+ * @Author: chenwei
+ */
+
+#include "curvefs/src/mds/schedule/operatorFactory.h"
+#include <cstdint>
+#include <memory>
+
+using ::std::chrono::steady_clock;
+
+namespace curvefs {
+namespace mds {
+namespace schedule {
+OperatorFactory operatorFactory;
```

Comment:
ditto

Reply:
Refactoring is more convenient in this way, no need to modify the previous code.